lib/shoryuken/environment_loader.rb

```diff
@@ -53,7 +53,9 @@ def config_file_options
     # https://github.com/phstc/shoryuken/blob/a81637d577b36c5cf245882733ea91a335b6602f/lib/shoryuken.rb#L82
     # Please delete this method afert next release (v2.0.12 or later)
     def initialize_aws
-      Shoryuken.logger.warn { "[DEPRECATION] aws in shoryuken.yml is deprecated. Please use configure_server and configure_client in your initializer"} unless Shoryuken.options[:aws].nil?
+      unless Shoryuken.options[:aws].to_h.empty?
+        Shoryuken.logger.warn { '[DEPRECATION] aws in shoryuken.yml is deprecated. Please use configure_server and configure_client in your initializer' }
+      end
```

Comment:
Sorry, I missed this https://github.com/phstc/shoryuken/blob/master/lib/shoryuken.rb#L29 .
Thank you for your fix.

I recommend use `configure_server` and `configure_client` in initializer. So, I add this `DEPRECATION` warning. 

Now, Shoryuken has multipe configure methods of aws.
First, worker side has two ways:

- Write aws information in `shoryuken.yml`
- Use `configure_client` block in initializer

Second, producer side has a few ways:

- Set environment variables, `AWS_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY`
- Use `~/.aws/credentials`
- Use Instance profile
- Use `configure_server` block in initializer

I thought this situation was complicated.
And I thought that setting aws in separate place is not good.

But, It may not be bad for multiple methods to exist, if we can control. 
What do you think?